package/MDAnalysis/analysis/encore/similarity.py

```diff
@@ -76,17 +76,16 @@
     >>> import MDAnalysis.analysis.encore as encore
     >>> from MDAnalysis.tests.datafiles import PSF, DCD, DCD2
 
-
 To calculate the Harmonic Ensemble Similarity (:func:`hes`)
 two ensemble objects are first created and then used for calculation: ::
 
     >>> ens1 = Universe(PSF, DCD)
     >>> ens2 = Universe(PSF, DCD2)
-    >>> print encore.hes([ens1, ens2])
-    (array([[        0.        ,  38279683.95892926],
-            [ 38279683.95892926,         0.        ]]), None)
+    >>> HES, details = encore.hes([ens1, ens2])
+    >>> print HES
+    [[        0.        ,  38279683.95892926],
+    [ 38279683.95892926,         0.        ]]
```

Comment:
add a `.. versionchanged:: 1.0.0` with the change description – when the API changes we need to document it